src/components/PostDetails.tsx

```diff
@@ -1,106 +1,121 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Comment, CommentData } from '../types/Comment';
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+
+type Props = {
+  isCommentLoading: boolean;
+  postComments: Comment[];
+  selectedPost: Post | null;
+  errorsComment: boolean;
+  onDeleteComment: (commentId: number) => void;
+  onAddComment: (newComment: CommentData) => void;
+  isButtonAddLoading: boolean;
+  onShowCommentFormChange: (item: boolean) => void;
```

Comment:
The method name 'onShowCommentFormChange' does not follow the naming conventions for event handler functions. According to the conventions, the name of the method should be 'handleShowCommentFormChange'. This makes the code more readable and maintainable.